addons/mail/static/src/widgets/form_renderer/form_renderer.js

```diff
@@ -130,7 +130,7 @@ FormRenderer.include({
      */
     _renderNode(node) {
         if (node.tag === 'div' && node.attrs.class === 'oe_chatter') {
-            if (this._isFromFormViewDialog) {
+            if (this._isFromFormViewDialog && !this.$attachmentPreview) {
```

Comment:
We don't want to create a chatter when it is not going to be displayed.

Proper fix is likely to quick return in `_renderView` in `mail_enterprise` if not `this._hasChatter()` after `super`.